docs/try-viam/rover-resources/rover-tutorial-fragments.md

```diff
@@ -53,28 +54,13 @@ Typically, this would suggest that they should be configured as enable pins, but
 
 {{% /alert %}}
 
-* A wheeled Viam [base](/components/base/) with attributes:
-  * Right Motors: right
-  * Left Motors: left
-  * Wheel circumference (mm): 217
-  * Width (mm): 260
-  * Spin slip factor: 1
-  * Depends on: left, right, local.
-* A webcam [camera](/components/camera/) with video_path: video0 and depends on: local.
-* Renc [encoder](/components/encoder/) with board: local, pins "i": "re" and depends on: local.
-* Lenc [encoder](/components/encoder/) with board: local, pins "i": "le" and depends on: local.
-* An [accelerometer](/components/movement-sensor/) with the following configuration:
-  * Model: "accel-adxl345"
-  * Name: "accelerometer"
-  * Type: "movement_sensor"
-  * and attributes of "i2c_bus": "default_i2c_bus", "use_alternate_i2c_address": false, and "board": "local".
-  * Depends on: local.
-* A microphone, type: audio_input, with attributes "audio_path_pattern": "3a" and "debug": false.
+For information about how you would configure a component yourself if you weren't using the fragment, click the links on each component above.
+To see the configured pin numbers and other values specific to this fragment, [see the components on the configuration page](#see-the-components-on-the-configuration-page).
```

Comment:
But if you were to configure it yourself you wouldn't have them there, how about
```suggestion
To see the configured pin numbers and other values specific to this fragment, [see the Viam Rover fragment in the Viam app](https://app.viam.com/fragment?id=3e8e0e1c-f515-4eac-8307-b6c9de7cfb84).
```

Reply:
Ooh I didn't mean for people to actually configure things themselves, just as a hypothetical. I guess this works too though?